src/fclaw2d_convenience.h

```diff
@@ -27,6 +27,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define FCLAW2D_CONVENIENCE_H
 
 #include <forestclaw2d.h>
+#include <p4est.h>
 #include <p4est_connectivity.h>
```

Comment:
The p4est_connectivity.h inclusion is now redundant. There is the convention to include it anyway, but there is also the alternative to remove redundant includes. In this case, I'd rather remove it here.

Reply:
Alright, I removed the redundant includes in fclaw{2,3}d_convenience.h.